src/AddAnyFile.csproj

```diff
@@ -110,6 +110,7 @@
     <Content Include="Templates\.md.txt">
       <IncludeInVSIX>true</IncludeInVSIX>
     </Content>
+    <Resource Include="Templates\.xaml.txt" />
```

Comment:
Templates should be <Content> tags and have the Include in VISX set - see other templates for examples